protocol/contracts/leaser/src/leaser.rs

```diff
@@ -185,46 +193,121 @@ fn emit_status(next_customer: Option<Addr>) -> Emitter {
 mod test {
     use admin_contract::msg::{MigrationSpec, ProtocolContracts};
     use cosmwasm_std::Addr;
+    use currencies::test::LpnC;
+    use finance::{coin::Coin, duration::Duration, liability::Liability, percent::Percent};
+    use lease::api::{
+        open::{ConnectionParams, Ics20Channel, PositionSpecDTO},
+        MigrateMsg,
+    };
+    use platform::{contract::Code, response};
     use sdk::cosmwasm_std::testing::MockStorage;
 
-    use crate::{msg::ForceClose, state::leases::Leases, ContractError};
+    use crate::{
+        msg::{Config, ForceClose, InstantiateMsg, MaxLeases},
+        state::leases::Leases,
+        ContractError,
+    };
+
+    const MAX_LEASES: MaxLeases = 100_000;
+    const LEASE_VOID_CODE: Code = Code::unchecked(12);
 
     #[test]
     fn close_empty_protocol() {
-        let store = MockStorage::default();
-        assert!(super::try_close_protocol(
-            &store,
+        let mut store = MockStorage::default();
+        Config::new(Code::unchecked(10), dummy_instantiate_msg())
+            .store(&mut store)
+            .unwrap();
+        let resp = super::try_close_protocol(
+            &mut store,
+            LEASE_VOID_CODE,
+            MAX_LEASES,
+            migrate_msg(),
             |_| Ok(Addr::unchecked("Registry")),
             dummy_spec(),
-            ForceClose::No
+            ForceClose::No,
         )
-        .is_ok());
+        .unwrap();
+        let cw_resp = response::response_only_messages(resp);
+        let update_lpp_update_reserve_delete_protocol = 1 + 1 + 1;
```

Comment:
???
If there is meaning implied by the two additions, I feel like writing `3` and just explaining where it comes from with a comment would make sense as I don't understand it myself like this.
```suggestion
        let update_lpp_update_reserve_delete_protocol = 3;
```

Reply:
avoiding docs if the code may explain the stuff is what we want
here the name of the variable and its value convey the meaning

Reply:
With all due respect, that's not a solution. The moment a 4th contract appears are we going to append it? A variable name that is longer than 2-3 words usually is not properly named. There are cases where it's unavoidable, but this isn't it. Why are we "scared" to write comments?

Reply:
I am not "scared", the practice is one to prefer explanatory code than documentation.